package.json

```diff
@@ -3,14 +3,16 @@
   "version": "1.0.0-22",
   "description": "clojure.spec for Javascript",
   "main": "dist/js.spec.js",
+  "types": "types/index.d.ts",
   "scripts": {
     "build": "webpack --config webpack.config.js",
     "lint": "yarn run lint-src && yarn run lint-test",
     "lint-src": "eslint --config .eslintrc.json 'index.js' 'lib/**/*.js'",
     "lint-test": "eslint --config .eslintrc.test.json 'test/**/*.js'",
-    "prepare": "yarn run lint && yarn run build",
-    "test": "mocha --recursive --compilers js:babel-register test",
+    "prepare": "yarn run build",
+    "test": "yarn run lint && yarn run test-ts && mocha --recursive --compilers js:babel-register test",
```

Comment:
I saw that `.travis.yml` actually does `linting` before everything else, but I am actually more happy with this approach